src/System.Private.CoreLib/shared/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -444,7 +444,7 @@ private void HandleSatelliteMissing()
                 satAssemName += ", Version=" + _mediator.SatelliteContractVersion.ToString();
             }
 
-            byte[] token = _mediator.MainAssembly.GetName().GetPublicKeyToken();
+            byte[] token = _mediator.MainAssembly.GetName().GetPublicKeyToken()!;
```

Comment:
This looks like a bug. I do see anything that guarantees `GetPublicKeyToken` to be non-null here. This should be `byte[]?` and the next block of code should be under `if (token != null)`.

Reply:
Fixed.